### ISSUE TITLE
Qualitative Data – Two Categories

### ISSUE THREAD

**Issue by Seulah-K** [Seulah-K/statistics_study#16]
*Summarized the lecture 'Inferential and Predictive Statistics for Business' from the University of Illinois Urbana-Champaign on Coursera.

How can we make regression models with categorical variables?

Categorical variables cannot be used directly into the model. 
We need to change them a number, called a dummy variable.
A dummy variable always has a value of either 0 or 1. 
For example, if the mother smoked during the pregnancy, the dummy variable will be 1 and 0, otherwise to represent yes or no.

> Example

Let's say that there is a study that shows pay inequity between male and female nurses in the United States.
To check the correlation between gender and pay, the researcher collected random salaries of 120 male and 120 female nurses. 
And, the researcher also takes the length of experience into consideration as an explanatory factor for the salary.

- y : salary
- X1 : experience (recorded in months)
- X2 : gender (categorical)
- Set the dummy:  1 for the male nurse, 0 for otherwise (female)

In multiple regression, if the independent variable is closely correlated with another one, we should get rid of one of the variables not to violate the independence assumption.

- Adjusted R square : 0.94
- P-value of months of experience : 1.2737E-126
- P-value of gender : 4.041E-112
- ŷ = 61149.40 + 82.3X1 + 4667.7X2 ( X1 : months of experience, X2 : gender)
- for female nurses : X2 = 0
- for male nurses  : X2 = 1

To sum up, in this example, we can expect how much individual variable affects the predictive result.
